pandas/core/frame.py

```diff
@@ -8029,6 +8029,35 @@ def idxmax(self, axis=0, skipna=True) -> Series:
         Notes
         -----
         This method is the DataFrame version of ``ndarray.argmax``.
+
+        Examples
+        --------
+        Consider dataset containing food consumption in Argentina.
+
+        >>> df = pd.DataFrame({'consumption': [10.51, 103.11, 55.48],
+        ...                    'co2_emission': [37.2, 19.66, 1712]},
```

Comment:
...                    'co2_emissions': [37.2, 19.66, 1712]},

(this will have to change in the other parts too)